build/devenv/environment.go

```diff
@@ -241,23 +250,80 @@ func NewEnvironment() (in *Cfg, err error) {
 	timeTrack.Record("[infra] deployed CL nodes")
 	timeTrack.Record("[changeset] deployed product contracts")
 
-	_, err = services.NewExecutor(in.Executor)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create executor service: %w", err)
-	}
+	if in.Mode == CL { //nolint:nestif // large block needed for clarity, refactor as a cl node component later
+		clChainConfigs := make([]string, 0)
+		clChainConfigs = append(clChainConfigs, CommonCLNodesConfig)
+		for i, impl := range impls {
+			var clChainConfig string
+			clChainConfig, err = impl.ConfigureNodes(ctx, in.Blockchains[i])
+			if err != nil {
+				return nil, fmt.Errorf("failed to deploy local networks: %w", err)
+			}
+			clChainConfigs = append(clChainConfigs, clChainConfig)
+		}
+		allConfigs := strings.Join(clChainConfigs, "\n")
+		for _, nodeSpec := range in.NodeSets[0].NodeSpecs {
+			nodeSpec.Node.TestConfigOverrides = allConfigs
+		}
+		Plog.Info().Msg("Nodes network configuration is generated")
 
-	for _, ver := range in.Verifier {
-		ver.ConfigFilePath = fmt.Sprintf("/app/cmd/verifier/testconfig/%s/verifier-%d.toml", ver.CommitteeName, ver.NodeIndex+1)
-		ver.SigningKey = cciptestinterfaces.XXXNewVerifierPrivateKey(ver.CommitteeName, ver.NodeIndex)
-		_, err = services.NewVerifier(ver)
+		timeTrack.Record("[changeset] configured nodes network")
+		_, err = ns.NewSharedDBNodeSet(in.NodeSets[0], nil)
 		if err != nil {
-			return nil, fmt.Errorf("failed to create verifier service: %w", err)
+			return nil, fmt.Errorf("failed to create new shared db node set: %w", err)
+		}
+
+		// Fund nodes...
+		for i, impl := range impls {
+			if err = impl.FundNodes(ctx, in.NodeSets, in.Blockchains[i], big.NewInt(1), big.NewInt(5)); err != nil {
+				return nil, err
+			}
+		}
+
+		// Configured keys on CL nodes
+		clClients, err := clclient.New(in.NodeSets[0].Out.CLNodes)
+		if err != nil {
+			return nil, fmt.Errorf("failed to connect CL node clients")
+		}
+
+		// transforming Executor and Verifier configs to Jobs
+		for _, cc := range clClients {
+			// TODO: generate keys instead of hard coding them
+			// TODO: generation could be done by devenv, and imported into CL nodes here, or they could be
```

Comment:
It's better to generate the keys ourselves, easier to control and more explicit IMO, of course, if it won't require pulling another dependency.